src/api/queries/membership.graphql

```diff
@@ -0,0 +1,28 @@
+fragment BasicChannelFields on Channel {
```

Comment:
Let's call the file `memberships.graphql` to stay consistent with other files in the same directory